man/git-alias.md

```diff
@@ -6,13 +6,23 @@ git-alias(1) -- Define, search and show aliases
 `git-alias`  
 `git-alias` &lt;search-pattern&gt;  
 `git-alias` &lt;alias-name&gt; &lt;command&gt;  
+`git-alias` [--global] &lt;alias-name&gt; &lt;command&gt;
```

Comment:
It seems we missed a case that the scope parameter is used with `search-pattern`?